packages/react/ds/src/header/header.tsx

```diff
@@ -42,13 +66,33 @@ export function Header({ logoLink, languages, navLinks }: HeaderProps) {
         id="MenuContainer"
         className="gi-h-20 gi-justify-between gi-items-center gi-flex gi-bg-emerald-800 gi-relative gi-py-3 gi-px-4 sm:gi-px-8 sm:gi-py-4"
       >
-        <div className="gi-flex">
-          <a href={logoLink} className="xs:gi-block gi-hidden">
-            <GovieLogo />
+        <div className="gi-flex gi-items-center gi-gap-3">
+          <a href={logo.href} className="xs:gi-block gi-hidden">
+            {logo.image ? (
+              <img
+                className="gi-object-contain gi-h-12"
+                src="https://ds.blocks.gov.ie/_next/static/media/hero.01b46751.png"
```

Comment:
You should add the `{logo.image}` here as `src`

Reply:
fixed